### ISSUE TITLE
Provide capability to automatically re-check the status of a deployment

### ISSUE THREAD

**Issue by BobJWalker** [OctopusSolutionsEngineering/OctopusCopilot#15]
### Chat query

It's a bit annoying to have to constantly ask the chatbot for the status of the deployment every minute or so.  I'm not sure how feasible it is to have the extension / copilot do that for me.

Some ideas:
- Check the status of  [release number] to [environment name] for [project name] in [space name] every 30 seconds until finished
- Deploy [release number] to [environment name] for [project name] in [space name] and show the status every 30 seconds until finished

### Action

Basically, this is me being lazy.  Rather than asking for the status every so often, it'd be good to be able to get the status automatically.

**Comment by mcasperson** [comment]
This is a limitation of the Copilot chat interface. There is no facility to push content to the chat like you might in a platform like Slack - it is strictly a request/response model.

In theory the underlying platform could be extended to true chat interfaces like Slack, but this is beyond the scope of the this implementation.
